### ISSUE TITLE
Build on local machine

### ISSUE THREAD

**Issue by joaomello** [dotnet/docs#849]
Hi all!
I'm trying to build the docs on my machine, but docfx appears to only copy the images... and not generating the site. I'm using docfx 2.2.

![Image of server](http://i.imgur.com/cRCIRaX.png)

**Command:**

```
D:\oss\dotnet\core-docs>docfx .\docfx.json --serve
Info: No files are found with glob pattern corefx/src/**/ref/**/*.csproj,wcf/src/**/ref/**/*.csproj, excluding **/bin/**,**/obj/**, under directory "D:\oss\dotnet"
Info: No files are found with glob pattern corefx/src/**/src/**/*.csproj,wcf/src/**/src/**/*.csproj,coreclr/src/mscorlib/System.Private.CoreLib.csproj, excluding **/bin/**,**/obj/**, under directory "D:\oss\dotnet"
Warning: Unable to find matching resource docs.html.
Warning: No resource found for [docs.html].
Info: 5 plug-in(s) loaded.
Info: No files are found with glob pattern *.yml, excluding <none>, under directory "D:\oss\dotnet\core-docs\api_ref"
Info: No files are found with glob pattern apidoc/*.md, excluding <none>, under directory "D:\oss\dotnet\core-docs"
Info: [Build Document]Max parallelism is 4.
Warning: [Build Document]Unable to find matching resource docs.html.
Info: [Build Document]Markdown engine is dfm
Info: [Build Document.ResourceDocumentProcessor]Building 26 file(s) in ResourceDocumentProcessor(ValidateResourceMetadata)...
Info: [Build Document.ConceptualDocumentProcessor]Building 131 file(s) in ConceptualDocumentProcessor(BuildConceptualDocument=>ValidateConceptualDocumentMetadata=>CountWord)...
Info: [Build Document.TocDocumentProcessor]Building 2 file(s) in TocDocumentProcessor(BuildTocDocument)...
Info: [Build Document]0 external references found in 1 xref maps.
Warning: [Build Document.Apply Templates]There is no template processing document type(s): Conceptual,Toc
Info: [Build Document.Apply Templates]Applying templates to 159 model(s)...
Info: [Build Document]XRef map exported.
Info: Manifest file saved to D:\oss\dotnet\core-docs\_site\manifest.json.
Info: Completed building documents in 2990,0466 milliseconds.
Info: [Apply Theme]Theme is applied.
Serving "D:\oss\dotnet\core-docs\_site" on http://localhost:8080
```

Is possible to build the docs?

**Comment by BillWagner** [comment]
/cc @bradygaster 

The documentation will build (in that all the HTML files are generated from the markdown source.)  Brady (cc'ed) can provide more information on what does work, and how this will change moving forward. We know we need to provide a better experience in generating the final site in a developer environment.

**Comment by svick** [comment]
Related: Local docfx build is now much less useful https://github.com/dotnet/core-docs/issues/627

**Comment by mairaw** [comment]
Are you all experiencing this @BillWagner, @svick? I've updated my docfx to 2.2.2 today and ran the build and I'm still able to see the site. It's not pretty because of the lack of a template, but it builds the docs.

![image](https://cloud.githubusercontent.com/assets/12971179/17384956/5c6eff6e-5994-11e6-9de6-881e76225600.png)

**Comment by BillWagner** [comment]
It's better. It's building the docs, and the images are deployed to the _site directory (which wasn't happening with an earlier version)

**Comment by mairaw** [comment]
Yes, I've fixed the issue with the images last month. So not sure why @joaomello is experiencing this. @chenkennt any clues?

**Comment by chenkennt** [comment]
This is because the template for docs.microsoft.com has been moved out of DocFX to a separate repo. Unfortunately that repo is private so external user cannot build the docs. @DuncanmaMSFT , is it possible to make the template repo public so that external user can build the repo locally?

**Comment by mairaw** [comment]
But I thought customers could still run DocFx however they wouldn't get the template or a TOC. Isn't that the case? I haven't managed to use the PowerShell script yet since I got some errors. That one I was expecting not to work for the community. I'll be opening a feature request to enable this. If we want the community to help us with the docs, they should have access to the same tools that we have.

**Comment by chenkennt** [comment]
You can build the docs using DocFX default template: `docfx build -t default`. The only problem is the nav bar on the top doesn't display correctly and links on it are broken because that's designed specifically for docs.microsoft.com

**Comment by BillWagner** [comment]
Fixed in #906
